uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/ControllerConf.java

```diff
@@ -30,8 +33,12 @@
  */
 public class ControllerConf extends PropertiesConfiguration {
 
+  private static final String CONFIG_FILE = "config.file";
```

Comment:
Maybe use a more straight-forward name to indicate the content of the config file.